srcpkgs/gzdoom/patches/fix-i686.patch

```diff
@@ -1,24 +1,23 @@
 diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
-index 26c617e..e162638 100644
+index 911e35d..813858e 100644
 --- a/src/CMakeLists.txt
 +++ b/src/CMakeLists.txt
-@@ -1361,11 +1361,11 @@ endif()
- if( ZD_CMAKE_COMPILER_IS_GNUCXX_COMPATIBLE )
+@@ -1376,10 +1376,11 @@ install(DIRECTORY "${PROJECT_BINARY_DIR}/soundfonts" "${PROJECT_BINARY_DIR}/fm_b
+ 
+ if( DEM_CMAKE_COMPILER_IS_GNUCXX_COMPATIBLE )
  	# Need to enable intrinsics for these files.
- 	if( SSE_MATTERS )
--		set_property( SOURCE
-+		set_source_files_properties(
- 			common/rendering/polyrenderer/poly_all.cpp
- 			rendering/swrenderer/r_all.cpp
--			utility/palette.cpp
--			utility/x86.cpp
-+			common/utility/palette.cpp
-+			common/utility/x86.cpp
- 			APPEND_STRING PROPERTY COMPILE_FLAGS " -msse2 -mmmx" )
- 	endif()
+-	set_property( SOURCE
++        set_source_files_properties(
++                common/rendering/polyrenderer/poly_all.cpp
++		rendering/swrenderer/r_all.cpp
+ 		common/utility/palette.cpp
+ 		common/utility/x86.cpp
+-		rendering/swrenderer/r_all.cpp
+ 		APPEND_STRING PROPERTY COMPILE_FLAGS " ${SSE2_ENABLE}" )
```

Comment:
```suggestion
-		APPEND_STRING PROPERTY COMPILE_FLAGS " ${SSE2_ENABLE}" )
+ 		APPEND_STRING PROPERTY COMPILE_FLAGS " -msse2 -mfpmath=sse" )
```

Reply:
Alright, i make a little test with what you sent, thanks again !

If everything works out, i'm gonna re-edit the template to remove the comments and re-commit. It's currently just a test.

Reply:
alright i'm aware some things went wrong here, but i'm doing another test, this one it should be good.

Reply:
damn... i really hoped it would work... :(